benchmarks/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+
+const fs = require("fs");
+const Promise = require("bluebird");
```

Comment:
Still have `bluebird`. Is there a way we can CI this to avoid issues like this in the future?